tools/code_analyzer/op_dependency.cpp

```diff
@@ -710,11 +768,14 @@ class OpDependency : public ModulePass {
     }
   }
 
-  static std::shared_ptr<std::string> extractOpSchema(Value* V) {
+  static std::shared_ptr<std::string> extractOpSchema(
+      const std::string& contextualNamespace, Value* V) {
```

Comment:
note to self: `contextualNamespace` comes with the trailing `::` attached already